kanin/src/app.rs

```diff
@@ -43,6 +43,14 @@ impl App {
         Self::default()
     }
 
+    /// Creates a connection with default properties. This is meant to be used in conjunction with [App::run_with_connection].
+    ///
+    /// # Errors
+    /// If a connection to the AMQP broker could not be established.
+    pub async fn create_connection(amqp_addr: &str) -> Result<Connection> {
```

Comment:
Consider just naming it `connect`.

Aside from that, this feels like a strange function to have on `App` - should we make a `Connection` type in kanin that wraps lapin's `Connection` which could then be used in `run_with_connection`?

Reply:
Or perhaps simply re-export lapin's connection as `kanin::Connection`

Reply:
I thought about this too, but you still end up with the question of "What should the connection properties look like?" because you are essentially creating a connection for Kanin and less so for yourself.

Reply:
But you are making it for yourself - you need it for something else (e.g. the service clients). Can't people just call `Connection::connect(amqp_addr, ConnectionProperties::default()` by themselves? Or maybe just make this function into a free-standing function `kanin::connect`?